bigtable/client/row_range.h

```diff
@@ -0,0 +1,75 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef BIGTABLE_CLIENT_DATA_H_
+#define BIGTABLE_CLIENT_DATA_H_
+
+#include <string>
+
+namespace bigtable {
+inline namespace BIGTABLE_CLIENT_NS {
+
+class RowRange {
+ public:
+  RowRange(std::string begin, std::string end)
+      : start_(begin), limit_(end) {}
+
+  // Create different types of ranges.
+  static RowRange infinite_range();
```

Comment:
We have been following the Google Style Guide for naming conventions, these should be in `CamelCase`.  Sorry the document was unclear about this.